modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying architectures on the Agent-based installer 
+
+You can check which architecture your {product-title} agent installer cluster is using by viewing the version binary.
+
+.Prerequisites: 
+
+* You installed the OpenShift CLI (oc)
+* You have the openshift-installer binary
+
+.Procedure: 
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshift-install 4.15.0-ec.3
+built from commit e80d50ac8c9ab65833045d648a01eb1a0a3a6d7d
+release image quay.io/openshift-release-dev/ocp-release@sha256:27d41f99e0784a2f596e773799983147b794df48d1f7536d51a95a60945b1931
+release architecture amd64
```

Comment:
nit, can we make up a commit and release sha? feels weird to have real values here?

Reply:
Yes, looking at that now!